tools/web_features/manifest.py

```diff
@@ -0,0 +1,249 @@
+#!/usr/bin/env python3
+
+import argparse
+import json
+import logging
+import os
+
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Any, List, Optional, Set
+
+from ..manifest.item import SupportFile
+from ..manifest.sourcefile import SourceFile
+from ..metadata.yaml.load import load_data_to_dict
+from ..web_features.web_feature_map import WebFeatureToTestsDirMapper, WebFeaturesMap
+from .. import localpaths
+from ..metadata.webfeatures.schema import WEB_FEATURES_YML_FILENAME, WebFeaturesFile
+
+"""
+This command generates a manifest file. This file contains a mapping of web
+features to manifest items. The data comes from the WEB_FEATURES.yml that are
+throughout the wpt source code. For more details about the file, check out RFC 163
+https://github.com/web-platform-tests/rfcs/pull/163.
+
+The file written is a JSON file. The format is
```

Comment:
I will create a new file called MANIFEST_SCHEMA.json (so that it won't be counted as a test file) https://github.com/web-platform-tests/wpt/blob/e7d1fb45fdf2ac5d5126e87cc5ebec7ef6c6b131/tools/manifest/sourcefile.py#L303

Reply:
Done